faker/providers/person/th_TH/__init__.py

```diff
@@ -62,29 +68,87 @@ class Provider(PersonProvider):
 
     prefixes = prefixes_female + prefixes_male
 
-    first_names = (
-        "พรชนก", "พัชชาพลอย", "ภีม", "กชพร", "พัทธพล", "สรัลพร", "จินจุฑา", "สรวุฒิ",
-        "สุวกิจ", "พริมา", "ดาริน", "พินท์ธุสร", "กุลนัน", "ณัฐชา", "ณัฐกฤตา", "สิทธิกร",
-        "วศิน", "อภิสรา", "ณัฐวรรณ", "ธันรดี", "นิรชา", "ธันยชนก", "กมลชนก", "จารุวรรณ",
-        "ผจงรักษ์", "ภัคจิรา", "ปัทธมพร", "สุวิจักษ์", "นพเก้า", "รัชชานนท์", "อธิศ",
-        "คุณาพร", "อริสรา", "ทัศวรรณ", "ชัยวัฒน์", "ภัณฑิรา", "ศุภสุตา", "พัชรพร",
-        "ภูเบศ",
-        "พัทธมน", "ชัญญา", "ปัณณวิชย์", "ชวิน", "ภาดา", "ชนิกานต์", "ณัฐวดี", "ชาลิสา",
-        "ประเมศฐ์", "ศุภสิทธิ์", "สิทธิวัต", "ธีธีช", "ญาดา", "เพ็ญพิชชา", "อนนต์",
-        "ชายฟ้า",
-        "ปวัน", "อัญญาร", "ญานิศา", "ภาคย์", "ชญานินท์", "ชยาภัทร", "จิตริน", "วัศยา",
-        "ปิติภัทร", "นิชากานต์", "ปริน", "ธนัชชา",)
+    # female first name
+    # http: // www.mamaexpert.com / posts / content - 1166
```

Comment:
Use actual URL, without the spaces. Also, add a brief description of the source.